packages/forma-36-react-components/src/components/TextField/__snapshots__/TextField.test.tsx.snap

```diff
@@ -51,13 +51,52 @@ exports[`renders the component with a copybutton 1`] = `
   margin-bottom: calc(1rem * (8 / 16));
 }
 
-.emotion-3 > button[disabled] {
-  pointer-events: none;
+.emotion-4 {
```

Comment:
Snapshots tests are back?

Reply:
This is from the `react-components` package, I only removed from the already migrated ones, but there are some components from there that are using the CopyButton

Reply:
Ah, I see. Makes sense now.